test/WebJobs.Script.Tests/Rpc/FunctionDispatcherTests.cs

```diff
@@ -227,6 +227,53 @@ public async void FunctionDispatcher_Restart_ErroredChannels_ExcceedsLimit()
             Assert.Equal(0, functionDispatcher.JobHostLanguageWorkerChannelManager.GetChannels().Count());
         }
 
+        [Fact]
+        public async void FunctionDispatcher_Restart_ErroredChannels_OnWorkerRestart_NotAffectedByLimit()
+        {
+            int expectedProcessCount = 2;
+            FunctionDispatcher functionDispatcher = (FunctionDispatcher)GetTestFunctionDispatcher(expectedProcessCount.ToString());
+            await functionDispatcher.InitializeAsync(GetTestFunctionsList(LanguageWorkerConstants.NodeLanguageWorkerName));
+
+            await WaitForJobhostWorkerChannelsToStartup(functionDispatcher, expectedProcessCount);
+            for (int restartCount = 0; restartCount < expectedProcessCount * 3; restartCount++)
+            {
+                foreach (var channel in functionDispatcher.JobHostLanguageWorkerChannelManager.GetChannels())
+                {
+                    TestLanguageWorkerChannel testWorkerChannel = channel as TestLanguageWorkerChannel;
+                    testWorkerChannel.RaiseWorkerRestart();
+                }
+
+                var finalChannelCount = await WaitForJobhostWorkerChannelsToStartup(functionDispatcher, expectedProcessCount);
+                Assert.Equal(expectedProcessCount, finalChannelCount);
+            }
+        }
+
+        [Fact]
+        public async void FunctionDispatcher_Restart_ErroredChannels_OnMixOfWorkerRestartAndErrorEvents()
+        {
```

Comment:
After we synced up offline and looking at the tests, you are right. We do not need test: FunctionDispatcher_Restart_ErroredChannels_OnMixOfWorkerRestartAndErrorEvents. please delete